### DISCUSSION TITLE
Rinkeby Testnet Eth

### DISCUSSION THREAD

**Discussion by Ya-Sin150** [smartcontractkit/full-blockchain-solidity-course-py#755] [Q&A] [answered]
I really need testnet eth. Rinkeby authentication faucet is really messing around with me and does not want to give me some eth. My address is below for anyone that wants to give me some eth.

0x5257086471ab48414aE854f461556A5b0B71CcDE

Thanks

**Comment by cromewar** [comment, marked as answer]
Sent: https://rinkeby.etherscan.io/tx/0x31244eae885fc47d9d2d663f0ce9a821a9d8e4b6369b8b151e8163ef78b16b3f

**Comment by Ya-Sin150** [comment]
Received Thanks
